packages/ckeditor5-table/src/commands/setheadercolumncommand.ts

```diff
@@ -92,4 +98,17 @@ export default class SetHeaderColumnCommand extends Command {
 			updateNumericAttribute( 'headingColumns', headingColumnsToSet, table, writer, 0 );
 		} );
 	}
+
+	/**
+	 * Checks if the table is a content table.
+	 */
+	private _isTableTypeContent( selectedCells: Array<Element> ): boolean {
+		const table = selectedCells[ 0 ].findAncestor( 'table' )!;
+
+		if ( table.getAttribute( 'tableType' ) == 'layout' ) {
+			return false;
+		}
+
+		return true;
```

Comment:
Can be simplified to 

```suggestion
		return table.getAttribute( 'tableType' ) !== 'layout';
```